src/core/api/base.go

```diff
@@ -189,9 +190,13 @@ func Init() error {
 
 	retentionMgr = retention.NewManager()
 
-	retentionLauncher = retention.NewLauncher(projectMgr, repository.Mgr, retentionMgr)
+	execMgr := task.NewExecutionManager()
```

Comment:
Use the global execution manager variable instead: https://github.com/goharbor/harbor/blob/master/src/pkg/task/execution.go#L33

The same to the task manager variable